modular_zubbers/code/modules/borgs/code/robot_defines.dm

```diff
@@ -8,6 +8,7 @@
 #define CYBORG_ICON_MED_WIDE_BUBBER 'modular_zubbers/code/modules/borgs/sprites/widerobot_med.dmi'
 #define CYBORG_ICON_MED_TALL_BUBBER 'modular_zubbers/code/modules/borgs/sprites/tallrobot_med.dmi'
 #define CYBORG_ICON_MED_LARGE_BUBBER 'modular_zubbers/code/modules/borgs/sprites/largerobot_med.dmi'
+#define CYBORG_ICON_MED_HAYDEE_BUBBER 'modular_zubbers/code/modules/borgs/sprites/boobrobot_med.dmi'
```

Comment:
```suggestion
#define CYBORG_ICON_MED_HAYDEE_BUBBER 'modular_zubbers/code/modules/borgs/sprites/boobrobot_med.dmi'
```
Can you rename all the dmi files to something else? Just call them Haydeez_med or whatever.

Reply:
Why?

Reply:
> Can you rename all the dmi files to something else? Just call them Haydeez_med or whatever.

Done. Changed boob to curve.